content/english/data_types/genomics_transcriptomics/services.md

```diff
@@ -16,7 +16,7 @@ The [SciLifeLab Genomics Platform](https://www.scilifelab.se/infrastructure/?cat
 
 The Clinical Genomics Stockholm facility has developed a WGS-based workflow for understanding genetic variation, especially in rare disease diagnostics settings. This workflow includes data generation, bioinformatics analysis and interpretation in our custom-developed clinical decision support system Scout.
 
-We believe this workflow may be useful also for COVID-19 projects aiming at understanding host susceptibility. Data generation is to 30x coverage using PCR-free libraries, bioinformatic analysis is comprehensive and includes calling of SNV, INDEL, CNV, SV, STR and UPDs. Variants are processed using a ranking model prior to made available for interpretation in Scout. This infrastructure has been applied to analysis of >6000 WGS samples.
+We believe this workflow may be useful also for COVID-19 projects aiming at understanding host susceptibility. Data generation is to 30x coverage using PCR-free liraries. The bioinformatic analysis is comprehensive and includes calling of SNV, INDEL, CNV, SV, STR and UPDs. Variants are processed using a ranking model prior to being made available for interpretation in Scout. This infrastructure has been applied to analysis of >6000 WGS samples.
```

Comment:
"liraries"?

Reply:
How do I suggest an actual edit?

Reply:
Dunno. I just went to the diff and chose to edit the file and then made the change.

Reply:
https://github.com/ScilifelabDataCentre/covid-portal/pull/99#issuecomment-637542640

Reply:
I can't demo this now, as the line has changed. But this is the button that you're looking for ☝️ 

![image](https://user-images.githubusercontent.com/465550/83526583-ec2c9700-a4e6-11ea-8ea6-e2bec9247440.png)